sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -119,10 +139,10 @@ def _transmit(self, envelopes: List[TelemetryItem]) -> ExportResult:
                             error.message,
                             envelopes[error.index] if error.index is not None else "",
                         )
-                if resend_envelopes:
+                if self.storage and resend_envelopes:
                     envelopes_to_store = [x.as_dict()
                                           for x in resend_envelopes]
-                    self.storage.put(envelopes_to_store)
+                    self.storage.put(envelopes_to_store, 0)
```

Comment:
What's the difference from adding 0?

Reply:
Adding 0 just means we can attempt to export the failed telemetry immediately, as opposed to ones with the default value (60s). We want the ones here to be attempting retry immediately because usually the ones that result from a returned `206` are valid and something jus happened coincidentally (perhaps in the backend) that rejected only partial envelopes.

Reply:
Thanks